src/main/scala/com/fulcrumgenomics/sopt/util/MarkDownProcessor.scala

```diff
@@ -80,6 +83,9 @@ class MarkDownProcessor(lineLength: Int = 80, indentSize: Int = 2) {
   /** Converts a MarkDown document to HTML. */
   def toHtml(document: Node): String = this.htmlRenderer.render(document).trim
 
+  /** Removes all trailing whitespace, without considering (i.e. keeping) ANSI terminal/color codes. */
+  private def trim(string: String): String = this.trimRegex.replaceAllIn(string, "$1")
```

Comment:
I can't tell if it's significant, but replacing `.trim` with this means that leading whitespace will be retained.

I wonder if it wouldn't be better to write something more verbose that:

1. Finds the first and last _printable_ non-whitespace character in the string
2. Captures any number of non-printable non-whitespace characters after the first/last
3. Emits a string composed of the leading/trailing non-printable chars and the core of the string